packages/webapp/src/stories/Form/CompositionInputs/CompositionInputs.stories.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { useState } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import type { Meta, StoryObj } from '@storybook/react';
+import { componentDecorators } from '../../Pages/config/Decorators';
+import CompositionInputs from '../../../components/Form/CompositionInputs';
+import Button from '../../../components/Form/Button';
+import { ReactComponent as RatioOptionIcon } from '../../../assets/images/ratio-option.svg';
+
+const meta: Meta<typeof CompositionInputs> = {
+  title: 'Components/CompositionInput',
+  component: CompositionInputs,
+  args: {
+    unitName: 'unit',
+    unitOptions: [
+      { label: '%', value: 'percent' },
+      {
+        label: <RatioOptionIcon />,
+        value: 'ratio',
+      },
+    ],
+    inputsInfo: [
+      { name: 'n', label: 'Nitrogen (N)' },
+      { name: 'p', label: 'Phosphorous (P)' },
+      { name: 'k', label: 'Potassium (K)' },
+    ],
+  },
+  decorators: [...componentDecorators],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  render: (args) => {
+    const { control } = useForm({
+      mode: 'onChange',
+      defaultValues: {
+        composition: {
+          unit: {
+            label: '%',
+            value: 'percent',
```

Comment:
Is there a reason this can't just be {unit: 'percent'} without the label? Because I see the label is already passed in with unitOptions.

Reply:
I'm not sure how to pass `value` as a `value` prop to `ReactSelect`. What is your idea??

Reply:
value prop is of the same type as option right? So you can get the option that has value = option.value from the unit options array. One possibility: `value={unitOptions.find(option => option.value === value}`

Reply:
Not to simplify the expression but to simplify the form data values when submitting to backend, but it might not matter that much so i'll leave it up to you.

Reply:
Ah okay, I believe the request body will be something like:
```
{
  n: x,
  p: x,
  k: x,
  npk_unit: 'percent',
  ...
}
```
, so we will need to format the data anyways.
I'll leave it as is :)

Reply:
Actually, having an option object as unit causes a [problem](https://github.com/orgs/react-hook-form/discussions/7764), so I updated it to store string!